### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by ahgittin**
minor language tweaks

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -8,18 +8,19 @@ Clocker for [Apache Brooklyn](https://brooklyn.apache.org/) is a set of open sou
 
 ## What can you do with it?
 
-* You can easily deploy production grade Docker Swarms or Kubernetes clusters to a [range of clouds](http://brooklyn.apache.org/v/latest/ops/locations/index.html) (including [AWS](https://aws.amazon.com/), [Azure](https://azure.microsoft.com), [Google Cloud](https://cloud.google.com/), [IBM Softlayer](http://www.softlayer.com/), and [IBM BlueBox](https://www.blueboxcloud.com/).
-* You can manage and scale Swarms or clusters in real time.
+* Easily deploy production-grade Docker Swarms or Kubernetes clusters to a [range of clouds](http://brooklyn.apache.org/v/latest/ops/locations/index.html) including [AWS](https://aws.amazon.com/), [Azure](https://azure.microsoft.com), [Google Cloud](https://cloud.google.com/), [IBM Softlayer](http://www.softlayer.com/), and [IBM BlueBox](https://www.blueboxcloud.com/)
+* Manage, manually scale, auto-scale, and get alerts on Swarms or clusters in real time
+* Deploy Brooklyn blueprints into these clusters or hybrid container/physical environments
```
Comment (tbouron) [inline index.md:13]: I deliberately didn't include this because it is not true for the open source kubernetes implementation (only available in AMP)

Comment (ahgittin) [reply index.md:13]: elsewhere in docs it mentions that functionality which is why i put it in here.

do we have a page writing up brooklyn v amp for this purpose?  i missed that.  thing to do is maybe include this but also include a statement `(other Brooklyn extensions may be required; see LINK)`

Comment (tbouron) [reply index.md:13]: @ahgittin I don't think we have this sort of page. I'll merge as it is, we can correct this easily afterward.

**Comment by tbouron** [comment]
LGTM, merging.

**Pull request merged**